invenio_app_rdm/stats/templates/aggregations/aggr_file_download/os-v1/aggr-file-download-v1.json

```diff
@@ -0,0 +1,69 @@
+{
+  "index_patterns": ["__SEARCH_INDEX_PREFIX__stats-file-download-*"],
```

Comment:
Shouldn't `__SEARCH_INDEX_PREFIX__` be taken care of in the search/indexer/stats level? (i.e. make it aware of the prefix?) Right now we are centralizing that in invenio-search. Maybe we need to move it somewhere different to have it all in one place.

Reply:
This is actually [an `invenio-search` feature](https://github.com/inveniosoftware/invenio-search/blob/18f6ea3adfec2f8e3e551d787df9b08001903d4b/invenio_search/ext.py#L413-L423) 🤯, so it's already centralized.

Reply:
Thanks! 🤯 👁️‍🗨️ 👁️‍🗨️  (I'll retreat to process this)